data/ecosystem/vendors.yaml

```diff
@@ -202,6 +202,13 @@
   contact: ''
   oss: false
   commercial: true
+- name: Middleware
+  distribution: true
```

Comment:
Your docs are not talking about a middleware distribution of OpenTelemetry, can you point to it?

Reply:
Yes, definitely. Middleware's [Agent](https://github.com/middleware-labs/mw-agent) is based on OTel collector. 

Reference 
MW Host Agent: https://github.com/middleware-labs/mw-agent/blob/master/cmd/host-agent/main.go#L365
MW Kubernetes Agent: https://github.com/middleware-labs/mw-agent/blob/master/cmd/kube-agent/main.go#L283